scripts/netContain/contivNet.sh

```diff
@@ -1,30 +1,36 @@
 #!/bin/bash
 #Initialize complete contiv container. Start OVS and Net Plugin
 
+cstore="$CONTIV_ETCD"
+vtep_ip="$VTEP_IP"
+vlan_if="$VLAN_IF"
+
+set -euo pipefail
+
 reinit=false
 plugin="docker"
-vtep_ip="$VTEP_IP"
-fwd_mode="bridge"
-cstore="$CONTIV_ETCD"
-cmode="bridge"
 netmaster=false
 netplugin=true
```

Comment:
Default is that it is run as netplugin. Only if we choose to run as netmaster, with the -m parameter, I am setting it to false in which case we will only run as netmaster.